lib/core/msdf-sys/src/lib.rs

```diff
@@ -0,0 +1,163 @@
+extern crate wasm_bindgen;
+#[macro_use] extern crate slice_as_array;
+
+use wasm_bindgen::prelude::*;
+use js_sys::*;
+
+const MAX_MSDF_SIZE : usize = 32;
+pub const MSDF_CHANNELS_COUNT : usize = 3;
+const MSDF_DATA_SIZE : usize = MAX_MSDF_SIZE*MAX_MSDF_SIZE*MSDF_CHANNELS_COUNT;
+
+pub struct FontHandle {
+    handle: JsValue
+}
+
+pub struct MSDF {
+    pub width  : usize,
+    pub height : usize,
+    pub data   : [f32;MSDF_DATA_SIZE]
+}
+
+pub struct MSDFParameters {
+    width                         : usize,
+    height                        : usize,
+    edge_coloring_angle_threshold : f64,
+    range                         : f64,
+    scale                         : (f64, f64),
+    translate                     : (f64, f64),
```

Comment:
Lets not use tuples. They are not descriptive and accessing elements by `.0` and `.1` is hacky.